src/controllers/group.js

```diff
@@ -0,0 +1,25 @@
+import GroupModel from '../models/group';
+
+const Group = {
+  register: async (req, res) => {
+    const newGroup = new GroupModel(
+      {
+        inviteCode: req.body.inviteCode,
```

Comment:
Invitecode is not given by the request.

Reply:
This was just code transferred from whatever was in main, I didn't add this.